src/Compilers/CSharp/Portable/Lowering/SynthesizedMethodBaseSymbol.cs

```diff
@@ -127,8 +130,8 @@ private ImmutableArray<ParameterSymbol> MakeParameters()
                     ordinal++,
                     p.RefKind,
                     p.Name,
-                    attributes: inheritAttributes ? p.GetAttributes() : default,
-                    hasEnumeratorCancellationAttribute: inheritAttributes && p.IsSourceParameterWithEnumeratorCancellationAttribute()));
+                    p.RefCustomModifiers,
```

Comment:
>p.RefCustomModifiers [](start = 20, length = 20)

Should these be substituted? It looks like we didn't pass these before. Do we have a test with a scenario where this change makes a difference?